kratos/processes/levelset_convection_process.h

```diff
@@ -840,6 +836,26 @@ class LevelSetConvectionProcess
         mEvaluateLimiter = (mIsBfecc || mElementRequiresLimiter) ? true : false;
     }
 
+    std::string GetConvectionElementRegisteredName(Parameters ThisParameters)
+    {
+        // Convection element formulation settings
+        std::string element_type = ThisParameters["element_type"].GetString();
+        const auto element_list = GetConvectionElementsList();
+        if (std::find(element_list.begin(), element_list.end(), element_type) == element_list.end()) {
+            KRATOS_INFO("") << "Specified \'" << element_type << "\' is not in the available elements list. " <<
+            "Attempting to use custom specified element." << std::endl;
```

Comment:
```suggestion
            KRATOS_INFO("") << "Specified \'" << element_type << "\' is not in the available elements list. Available elements are: \n\t- \'levelset_convection_supg\'\n\t- \'levelset_convection_algebraic_stabilization\'" <<     "\nAttempting to use custom specified element." << std::endl;
```
I mean this, so newbies (and users of the binary version) know which are the available elements without requiring to check the code.

Reply:
Not here...